File: iminling-core/src/main/java/com/iminling/core/filter/impl/DefaultLoginFilter.java
package com.iminling.core.filter.impl;

import com.iminling.core.filter.LoginFilter;
import org.springframework.web.method.HandlerMethod;

import javax.servlet.http.HttpServletRequest;

/**
 * 默认login实现
 * @author yslao@outlook.com
 */
public class DefaultLoginFilter implements LoginFilter {

    @Override
    public void doFilter(HandlerMethod handlerMethod, HttpServletRequest request) {
    }

    @Override
    public int getOrder() {
        return -99999;
    }
}

File: iminling-core/src/main/kotlin/com/iminling/core/config/filter/impl/DefaultLoginFilter.kt
package com.iminling.core.config.filter.impl

import com.iminling.core.config.filter.LoginFilter
import org.springframework.web.method.HandlerMethod
import javax.servlet.http.HttpServletRequest

/**
 * @author  yslao@outlook.com
 * @since  2021/11/28
 */
class DefaultLoginFilter: LoginFilter {
    override fun doFilter(handlerMethod: HandlerMethod, request: HttpServletRequest) {

    }

    override fun getOrder(): Int {
        return 1
    }
}